docs/platforms/javascript/common/configuration/integrations/openfeature.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: OpenFeature
+description: "Attaches recent OpenFeature feature flag evaluations to error event context."
+notSupported:
+  - javascript.aws-lambda
+  - javascript.azure-functions
+  - javascript.bun
+  - javascript.capacitor
+  - javascript.cloudflare
+  - javascript.connect
+  - javascript.cordova
+  - javascript.deno
+  - javascript.electron
+  - javascript.express
+  - javascript.fastify
+  - javascript.gcp-functions
+  - javascript.hapi
+  - javascript.koa
+  - javascript.nestjs
+  - javascript.nodejs
+  - javascript.wasm
+---
+
+<Alert level="info">
+
+This integration only works inside a browser environment.
+
+</Alert>
+
+_Import name: `Sentry.openFeatureIntegration` and `Sentry.OpenFeatureIntegrationHook`_
+
+The [OpenFeature](https://https://openfeature.dev/) integration tracks feature flag evaluations produced by the OpenFeature SDK. These evaluations are held in memory and, in the event an error occurs, sent to Sentry for review and analysis. **At the moment, we only support boolean flag evaluations.**
+
+```JavaScript
+import * as Sentry from '@sentry/browser';
+import { OpenFeature } from '@openfeature/web-sdk';
+
+Sentry.init({integrations: [Sentry.openFeatureIntegration()]});
+
+const client = OpenFeature.getClient();
+client.addHooks(new OpenFeatureIntegrationHook());
```

Comment:
More of a review to be done here: https://github.com/getsentry/sentry-javascript/pull/14326/

But could the integration itself do this? Since it's a "global" function that we're calling we could just auto register and avoid having the user do this bit of code?

Reply:
Not without taking on a dependency.  We either ask them to pass the client to us at integration init time (which may or may not be desired depending on their code's structure) or we ask them to register the hook explicitly.  Just my two cents.  Could be more options I don't know about.

Reply:
should this be `Sentry.OpenFeatureIntegrationHook`?

Reply:
Yes thank you!

Reply:
> Not without taking on a dependency. 

I see so we're not shipping a `@sentry/openfeature` here and this is the way?

It's not the easist set up but it does make our lives much easier for now at least (no additional package requirements)

Reply:
Yeah its just a browser integration so no extra install steps.  I agree its not an ideal setup.  But I don't think its unexpected setup friction either.

If one day the flag buffer is a property on the scope we could remove the integration step all together and just have people register the hook.  Which would be a one line change on the customer's side rather than two.

Reply:
Talking about the integration init here being removed: https://github.com/getsentry/sentry-docs/pull/11837/files#diff-9f766b64b39c2a17b915efb1536f14ed4dede9c406a6e9fe920f1f1110ba7bcfR38